static/src/javascripts/projects/commercial/modules/cmp/cmp.js

```diff
@@ -0,0 +1,216 @@
+// @flow
+import { log } from 'commercial/modules/cmp/log';
+import { CmpStore } from 'commercial/modules/cmp/store';
+import { getCookie } from 'lib/cookies';
+import { vendorList as globalVendorList } from 'commercial/modules/cmp/vendorlist';
+import {
+    defaultConfig,
+    CMP_GLOBAL_NAME,
+    CMP_ID,
+    CMP_VERSION,
+    COOKIE_VERSION,
+    COOKIE_NAME,
+} from 'commercial/modules/cmp/cmp-env';
+import { encodeVendorConsentData } from 'commercial/modules/cmp/cookie';
+
+import type { CmpConfig } from 'commercial/modules/cmp/types';
+
+const readConsentCookie = (cookieName: string): boolean | null => {
```

Comment:
I feel that this is redundant with the function available from common/modules/commercial/ad-prefs.lib.js

If you use the API provided by ad-prefs.lib.js, you don't need to worry about the cookie yourself :)